chain-tx-validation/src/lib.rs

```diff
@@ -359,7 +359,7 @@ pub fn verify_unbonding(
     account.unbond(
         maintx.value,
         extra_info.min_fee_computed.to_coin(),
-        extra_info.previous_block_time + i64::from(extra_info.unbonding_period),
+        extra_info.previous_block_time + u64::from(extra_info.unbonding_period),
```

Comment:
perhaps use the type alias:

```suggestion
        extra_info.previous_block_time + Timespec::from(extra_info.unbonding_period),
```

Reply:
Done